### PR TITLE
Add Azure related runtime dependencies

### PR DESCRIPTION
**Pull request by anusudarsan**

### CODE REVIEW

**Comment by electrum** [comment]
Did you verify the output jar (use `jar -tf`)? I think you need to remove the exclusions as well.

**Comment by anusudarsan** [comment]
Ya removed.
jar -tf target/hadoop-apache-3.2.0-7-SNAPSHOT.jar | grep okio
okio/

and
jar -tf target/hadoop-apache-3.2.0-7-SNAPSHOT.jar | grep okhttp
io/prestosql/hadoop/$internal/com/squareup/okhttp/

File: pom.xml [hunk 1]
```diff
@@ -551,6 +543,14 @@
                                     <pattern>org.slf4j</pattern>
                                     <shadedPattern>${shadeBase}.org.slf4j</shadedPattern>
                                 </relocation>
+                                <relocation>
+                                    <pattern>com.squareup.okhttp</pattern>
+                                    <shadedPattern>${shadeBase}.com.squareup.okhttp</shadedPattern>
+                                </relocation>
+                                <relocation>
+                                    <pattern>com.squareup.okio</pattern>
```
Comment (electrum) [inline pom.xml]: These need to be just `okio` to match the Java package name.

Comment (anusudarsan) [reply pom.xml]: done

**Pull request merged**